plugins/atlas/skills/xmlanawsbuilder/SKILL.md

```
@@ -0,0 +1,298 @@
+| ----------------------------- | ------------------------------------- |
+| `-x` / `--xml`                | Top-level XML card path (required)    |
+| `-v` / `--verbose`            | Print debug info                      |
+| `-m` / `--minimizerAlgo`      | `Minuit2` (default) or `Minuit`       |
+| `-s` / `--minimizerStrategy`  | 0, 1 (default), or 2                  |
+| `-t` / `--minimizerTolerance` | Default 1e-3                          |
+| `-b` / `--binned`             | Fit to binned (pseudo-binned) dataset |
+| `-o` / `--plotOption`         | Plot options (e.g. `logy`, `rebin10`) |
+
+## Three-Level XML Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)